support/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM node:18-alpine AS base
+WORKDIR /opt/isomercms-backend
+
+RUN apk update && \
+    apk add --no-cache bash && \
+    apk add git && \
+    apk add openssh-client
+
+RUN adduser -u 900 webapp -D -h /home/webapp -s /bin/sh
+USER webapp
+
+COPY ../src ./src
+COPY ../common ./common
+COPY ../support ./support
+COPY ../package.json . 
+COPY ../package-lock.json .
+COPY ../scripts ./scripts
+COPY ../tsconfig.json .
```

Comment:
change 1 - we selectively copy rather than just `COPY . .`; we should be able to omit `COPY ../src ./src` if we are rigorous about separation of concerns but for now this is ok. build times are not impacted too much cos both jobs run in parallel